modules/cnv-creating-vm-wizard-web.adoc

```diff
@@ -5,13 +5,13 @@
 [id="cnv-creating-vm-wizard-web_{context}"]
 = Running the virtual machine wizard to create a virtual machine
 
-The web console features an interactive wizard that guides you through *Basic Settings*, *Networking*, and *Storage* screens to simplify the process of creating virtual machines. All required fields are marked by a `*`. The wizard prevents you from moving to the next screen until the required fields have been completed.
+The web console features an interactive wizard that guides you through *General*, *Networking*, *Storage*, *Advanced*, and *Review* steps to simplify the process of creating virtual machines. All required fields are marked by a `*`. When the required fields are completed, you can review and create your virtual machine.
 
 NICs and storage disks can be created and attached to virtual machines after they have been created.
```

Comment:
Please define acronyms on first use in each module: `Network Interface Cards (NICs)`

Reply:
Expanded NIC to Network Interface Card followed by the acronym in brackets: Network Interface Cards (NICs)